docs/2_3_common_states.adoc

```diff
@@ -532,17 +532,17 @@ a|
 * Compute
   - latexmath:[\mathbf{f}_{\mathit{disc}}] if <<providesEvaluateDiscreteStates>> is `true` and latexmath:[\mathbf{f}_{\mathit{disc}}] (<<fmi3EvaluateDiscreteStates>>) was not called within this super-dense time, or
   - latexmath:[\mathbf{f}_{\mathit{event}}] if <<providesEvaluateDiscreteStates>> is `false` and latexmath:[\mathbf{f}_{\mathit{event}}] was not called within this super-dense time.
-* latexmath:[(\mathbf{T}_\mathit{interval}) := \mathbf{f}_{\mathit{update}}({}^\bullet\mathbf{x}_{c+d}, \mathbf{u}_{c+d}, \mathbf{p}, {}^\bullet\mathbf{b}, \mathbf{t})]
+* latexmath:[(T_{\mathit{start}}) := \mathbf{f}_{\mathit{update}}({}^\bullet\mathbf{x}_{c+d}, \mathbf{u}_{c+d}, \mathbf{p}, {}^\bullet\mathbf{b}, t)]
```

Comment:
was the change from interval to start intentional?

**T**_inteval is bold it we can have one for each changing clock.
However this is retrieved with getInteval function should it be added to the function influencing equations?

We miss a function
* latexmath:[(T_{\mathit{next}}) := \mathbf{f}_{\mathit{update}}({}^\bullet\mathbf{x}_{c+d}, \mathbf{u}_{c+d}, \mathbf{p}, {}^\bullet\mathbf{b}, t)]

Reply:
Thanks for catching the unintentional interval->start change (I did chase it as I did see it once, but did not find it again and thought it was gone...), T now bold (also in lines 520 and 522 - please check!), added missing function.